ds-collector/rust-commands/collect-info.rs

```diff
@@ -68,6 +68,7 @@ fn main() {
         dse_bin_dir: &env::var("dse_bin_dir").unwrap_or("".to_string()),
         dt_opts: &env::var("dt_opts").unwrap_or("".to_string()),
         solr_data_dir: &env::var("solr_data_dir").unwrap_or("".to_string()),
+        script_directory: &env::var("script_directory").unwrap_or(".".to_string()),
```

Comment:
On nodes $base_dir == $script_directory and the collect-info binary is only ever called on nodes.

This constraint is a bit of an issue when people want to manually run in client-mode (`-C`) on the node. (They have to know that the collector has to be extracted to the /tmp/datastax folder.) But we don't want to encourage users to do that, so i'd rather not be trying to incentivise it.

Even (/When) we want to introduce the script_directory variable, we need to take a general look at the separation between the two. For example one of the commands is `{base_dir}/dstat` which becomes wrong.

Reply:
okay I think I followed what you were saying here, the design of this overall is not straight-forward :D